apps/drec-api/src/pods/device/device.service.ts

```diff
@@ -123,6 +135,32 @@ export class DeviceService {
     return !!this.getFuelTypes().find((fuel) => fuel.code === fuelType);
   }
 
+  groupDevices(
+    orderFilterDto: DeviceGroupByDTO,
+    devices: Device[],
+  ): DeviceDTO[][] {
+    const { orderBy } = orderFilterDto;
+    const orderByRules: DeviceOrderBy[] = !Array.isArray(orderBy)
+      ? [orderBy]
+      : orderBy;
```

Comment:
You can flip those and get rid of the negate:
` Array.isArray(orderBy) ? orderBy : [orderBy];`
 
 Makes it easier to read (at least for me).